src/components/navigation/problem-tab-content.tsx

```diff
@@ -89,10 +89,11 @@ export const ProblemTabContent: React.FC<IProps>
         {isTeacher && showSolutionsSwitch &&
           <SolutionsButton onClick={handleToggleSolutions} isToggled={showTeacherContent} />}
       </div>
-      <div className="problem-panel">
+      <div className="problem-panels-container">
         {sections?.map((section) => {
           return (
-            <TabPanel key={`section-${section.type}`} data-focus-section={section.type}>
+            <TabPanel key={`section-${section.type}`} data-focus-section={section.type}
+                className={["react-tabs__tab-panel", "problem-panel-tab-panel"]}>
```

Comment:
Why use a class name with a double-understore like that?  Does it have a specific meaning?

Reply:
Yeah, that is the default class name that is added by the react-tabs when you don't specify your own. So when explicitly setting the class names it seems good to keep it included incase there are some generic react-tabs styles.